types/index.d.ts

```diff
@@ -784,6 +784,12 @@ declare namespace RamdaAdjunct {
         */
         allSettledP<T>(iterable: Iterable<T>): Promise<Array<SettledPromise<T>>>;
 
+        /**
+         * anyP accepts an iterable of promises and returns a promise that is fulfilled by the first given promise to be fulfilled,
+         * or rejected with an array of rejection reasons if all of the given promises are rejected.
+         */
+        anyP<T>(iterable: Iterable<T>): Promise<T>;
```

Comment:
We're missing tests for typings. You can learn more about how to do them here: https://github.com/char0n/ramda-adjunct/tree/master/types/test

Reply:
Thank you, I added tests for typings